learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -248,63 +267,67 @@ Accessing Scene Nodes
-Dragging from the Scene Tree and dropping into the canvas, by default, will ask you to *call a method* (sometimes referred to as *member function*) on
-this node. 
+Dragging from the Scene Tree and dropping into the canvas, by default, will ask you to *call a method* (sometimes referred to as *member function*) on this node. 
 
 .. image:: img/visual_script19.png
 
+
 While accessing properties is desired in most cases (more on that below), sometimes *calling methods* can be useful too.
 Methods execute specific actions on objects. In the above case, the mouse pointer can be warped to a position in local
 coordinates to the control. Another common use case is queueing a node for deletion, which is done with the *queue_free* method.
 
 .. image:: img/visual_script20.png
 
+
 Care must be taken that this only works if the scene being edited contains your *Visual Script* in one of the nodes! Otherwise, a warning will be shown.
 
+
 Accessing Scene Node Properties
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This will be by far the most common way to *Scene Nodes* in Visual Scripting. Simply select a *Scene Node* from the *Scene Tree*,
-go the the inspector, find *The Name* of the property you want to edit (hint, *not* the value!) and drag it to the canvas:
+This will be by far the most common way to edit *Scene Nodes* in Visual Scripting. Simply select a *Scene Node* from the *Scene Tree*, go to the inspector, find *The Name* of the property you want to edit (hint, *not* the value!) and drag it to the canvas:
 
 .. image:: img/visual_script21.png
 
+
 The result is that this value can be changed from your script by writing to a *Data Port*.
 
 If instead reading this value is desired, just drag the node again but hold the *Control* key (or Command on Mac). This will create a getter:
 
 .. image:: img/visual_script22.png
 
+
 In this case, the value can be read from a *Data Port*.
 
 
 Variables
 ~~~~~~~~~
 
-Variables are memory containers local the script, which can hold a value. This value can be read from any of the functions of the script, or
-from other scripts via the method described in the previous step.
+Variables are memory containers local to the script, which can hold a value. This value can be read from any of the functions of the script, or from other scripts via the method described in the previous step.
 
-Adding a Variable is simple, just push the "Plus" Button on the *Variables* section of the members panel. Afterwards, doubleclick the
-new variable to rename it:
+Adding a Variable is simple, just push the "Plus" Button on the *Variables* section of the members panel. Afterwards, doubleclick the new variable to rename it:
 
 .. image:: img/visual_script23.png
 
-Right clicking the variable allows to configure the type, as well as other properties:
+
+Right clicking the variable allows you to configure the type, as well as other properties:
```

Comment:
Right-clicking

The image will show the properties:
"allows you to configure its properties:"